install/kubernetes/cilium/templates/hubble/tls-cronjob/_job-spec.tpl

```diff
@@ -19,30 +19,94 @@ spec:
+                  {{- range $dns := .Values.hubble.tls.server.extraDnsNames }}
+                  - {{ $dns | quote }}
+                  {{- end }}
+                  {{- range $ip := .Values.hubble.tls.server.extraIpAddresses }}
+                  - {{ $ip | quote }}
+                  {{- end }}
+                  usage:
+                  - signing
+                  - key encipherment
+                  - server auth
+                  validity: {{ $certValidityStr }}
+                {{- if .Values.hubble.relay.enabled }}
+                - name: hubble-relay-client-certs
+                  namespace: {{ .Release.Namespace }}
+                  commonName: "*.hubble-relay.cilium.io"
+                  hosts:
+                  - "*.hubble-relay.cilium.io"
+                  usage:
+                  - signing
+                  - key encipherment
+                  - client auth
+                  validity: {{ $certValidityStr }}
+                {{- end }}
+                {{- if and .Values.hubble.relay.enabled .Values.hubble.relay.tls.server.enabled }}
+                - name: hubble-relay-server-certs
+                  namespace: {{ .Release.Namespace }}
+                  commonName: "*.hubble-relay.cilium.io"
+                  hosts:
+                  - "*.hubble-relay.cilium.io"
+                  {{- range $dns := .Values.hubble.relay.tls.server.extraDnsNames }}
+                  - {{ $dns | quote }}
+                  {{- end }}
+                  {{- range $ip := .Values.hubble.relay.tls.server.extraIpAddresses }}
+                  - {{ $ip | quote }}
+                  {{- end }}
+                  usage:
+                  - signing
+                  - key encipherment
+                  - server auth
+                  validity: {{ $certValidityStr }}
+                {{- end }}
+                {{- if and .Values.hubble.metrics.enabled .Values.hubble.metrics.tls.enabled }}
+                - name: hubble-metrics-server-certs
+                  namespace: {{ .Release.Namespace }}
+                  commonName: {{ list (.Values.cluster.name | replace "." "-") "hubble-metrics.cilium.io" | join "." }} | quote }}
+                  hosts:
+                  - {{ list (.Values.cluster.name | replace "." "-") "hubble-metrics.cilium.io" | join "." }} | quote }}
+                  {{- range $dns := .Values.hubble.metrics.tls.server.extraDnsNames }}
+                  - {{ $dns | quote }}
+                  {{- end }}
+                  {{- range $ip := .Values.hubble.metrics.tls.server.extraIpAddresses }}
+                  - {{ $ip | quote }}
+                  {{- end }}
+                  usage:
+                  - signing
+                  - key encipherment
+                  - server auth
+                  validity: {{ $certValidityStr }}
+                {{- end }}
+                {{- if and .Values.hubble.ui.enabled .Values.hubble.relay.enabled .Values.hubble.relay.tls.server.enabled }}
```

Comment:
Thanks for the drive-by fix! However, I think we only want to do that if mTLS is actually enabled on Hubble Relay's server:
```suggestion
                {{- if and .Values.hubble.ui.enabled .Values.hubble.relay.enabled .Values.hubble.relay.tls.server.enabled .Values.hubble.relay.tls.server.mtls }}
```

Reply:
Hmm, actually the `mtls` value is not taken into account both in case of [helm](https://github.com/giorio94/cilium/blob/mio%2Fcertgen-config/install/kubernetes/cilium/templates/hubble/tls-helm/ui-client-certs.yaml#L1) and [cert-manager](https://github.com/giorio94/cilium/blob/mio%2Fcertgen-config/install/kubernetes/cilium/templates/hubble/tls-certmanager/ui-client-certs.yaml#L1) generation. I'd probably defer that change to a separate PR. WDYT?